README.md

```diff
@@ -53,64 +70,27 @@ libraryDependencies += "com.google.auth" % "google-auth-library-oauth2-http" % "
 ```
 [//]: # ({x-version-update-end})
 
-## google-auth-library-credentials
-
-This artifact contains base classes and interfaces for Google credentials:
-- `Credentials`: base class for an authorized identity. Implementations of this class can be used to
-authorize your application
-- `RequestMetadataCallback`: interface for the callback that receives the result of the asynchronous
-`Credentials.getRequestMetadata(URI, Executor, RequestMetadataCallback)`
-- `ServiceAccountSigner`: interface for a service account signer. Implementations of this class are
-capable of signing byte arrays using the credentials associated to a Google Service Account
-
-## google-auth-library-appengine
-
-This artifact depends on the App Engine SDK (`appengine-api-1.0-sdk`) and should be used only by
-applications running on App Engine environments that use urlfetch. The `AppEngineCredentials` class
-allows you to authorize your App Engine application given an instance of
-[AppIdentityService][appengine-app-identity-service].
-
-Usage:
-
-```java
-import com.google.appengine.api.appidentity.AppIdentityService;
-import com.google.appengine.api.appidentity.AppIdentityServiceFactory;
-import com.google.auth.Credentials;
-import com.google.auth.appengine.AppEngineCredentials;
-
-AppIdentityService appIdentityService = AppIdentityServiceFactory.getAppIdentityService();
+## google-auth-library-oauth2-http
 
-Credentials credentials =
-    AppEngineCredentials.newBuilder()
-        .setScopes(...)
-        .setAppIdentityService(appIdentityService)
-        .build();
-```
+### Application Default Credentials
 
-**Important: `com.google.auth.appengine.AppEngineCredentials` is a separate class from
-`com.google.auth.oauth2.AppEngineCredentials`.**
+This library provides an implementation of [Application Default Credentials](https://cloud.google.com/docs/authentication/getting-started)
```

Comment:
ADC isn't mentioned on that page.  https://google.aip.dev/auth/4110 explains it at length.

I also asked for repairs to that page.

Reply:
You might also want to mention https://google.aip.dev/auth

Reply:
yep, thanks, originally I did not like the link either but forgot about it :)